packages/config/lib/index.js

```diff
@@ -130,6 +130,10 @@ module.exports = {
     return validateNoBreakingOptions(options, cfg, onWarning, onErr)
   },
 
+  validateNoBreakingConfigLaunchpad: (cfg, onWarning, onErr) => {
+    return validateNoBreakingOptions(breakingOptions.filter((option) => option.showInLaunchpad), cfg, onWarning, onErr)
+  },
```

Comment:
Just curious, are we able to use the existing API used to display the baseUrl warning message in the launchpad?

Reply:
We don't have the baseUrl warning message implemented yet in 10.0. I think Zach Williams is working on that in a different PR right now

Reply:
ohhh I thought that had merged